core-services/prow/02_config/openshift/api/_pluginconfig.yaml

```diff
@@ -18,3 +18,21 @@ plugins:
   openshift/api:
     plugins:
     - approve
+welcome:
```

Comment:
Don't forget to also add the plugin (either here or globally in openshift org):
```suggestion
    - approve
    - welcome
welcome:
```

Reply:
thanks, fixed